modules/apim-adapter/src/main/java/com/axway/apim/api/model/OutboundProfile.java

```diff
@@ -15,41 +15,44 @@
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 
 public class OutboundProfile extends Profile {
-	
+
 	protected static Logger LOG = LoggerFactory.getLogger(OutboundProfile.class);
-	
+
 	String routeType;
-	
-	@JsonDeserialize( using = PolicyDeserializer.class)
+
+	@JsonDeserialize(using = PolicyDeserializer.class)
 	Policy requestPolicy;
-	
-	@JsonDeserialize( using = PolicyDeserializer.class)
+
+	@JsonDeserialize(using = PolicyDeserializer.class)
 	Policy responsePolicy;
-	
-	@JsonDeserialize( using = PolicyDeserializer.class)
+
+	@JsonDeserialize(using = PolicyDeserializer.class)
 	Policy routePolicy;
-	
-	@JsonDeserialize( using = PolicyDeserializer.class)
+
+	@JsonDeserialize(using = PolicyDeserializer.class)
 	Policy faultHandlerPolicy;
-	
+
 	String authenticationProfile;
-	
+
 	List<Object> parameters = new ArrayList<Object>();
 
 	public OutboundProfile() throws AppException {
 		super();
 	}
 
 	public String getAuthenticationProfile() {
-		return this.authenticationProfile;
+		if (StringUtils.isBlank(authenticationProfile))
+			return "_default";
+		else
+			return this.authenticationProfile;
```

Comment:
Also here it would be great to have a comment that explains the reason, why this is required.

Reply:
done